src/OrchardCore.Modules/OrchardCore.Contents/Recipes/ContentStep.cs

```diff
@@ -41,6 +41,11 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                     // Initializes the Id as it could be interpreted as an updated object when added back to YesSql
                     contentItem.Id = 0;
                     await _contentManager.CreateAsync(contentItem);
+                    
+                    // Overwrite ModifiedUtc & PublishedUtc values that handlers have changes
+                    // Should not be necessary if IContentManager had an Import method
+                    contentItem.ModifiedUtc = contentItem.ModifiedUtc;
+                    contentItem.PublishedUtc = contentItem.PublishedUtc;
```

Comment:
@sebastienros I'm not sure how these lines will affect on the content item after it has been store in the handler?!!